drivers/gpu/drm/vc4/vc4_dpi.c

```diff
@@ -205,6 +207,11 @@ static void vc4_dpi_encoder_enable(struct drm_encoder *encoder)
 			}
 		}
 
+		if (dpi->rgb_order_override) {
```

Comment:
Are we happy that it's never necessary to force DPI_ORDER_RGB?

Reply:
All the RGB order formats are available via MEDIA_BUS_FMT defines, hence I saw no reason to override it.
I did think about using -1 as "not overridden", but it seemed overkill.